pipelines/RunIsrForCrosstalkSources.yaml

```diff
@@ -0,0 +1,22 @@
+description: Prepare overscanSources (for ISR/interchip crosstalk).
+instrument: lsst.obs.decam.DarkEnergyCamera
+tasks:
+  overscan:
+    class: lsst.ip.isr.isrTask.IsrTask
+    config:
+      connections.ccdExposure: 'raw'
+      connections.outputExposure: 'oscanRaw'
```

Comment:
Prefer to use the full name `overscanRaw` unless there is a strong case for shortening it